source/elements/oneMKL/source/domains/dft/config_params/data_layouts.rst

```diff
@@ -0,0 +1,299 @@
+  this leads to some data padding being required in forward domain if unit
+  strides are used along the last dimension in forward and backward domains.
+
+.. _onemkl_dft_io_strides_deprecated:
+
+.. rubric:: Configuring strides for input and output data [deprecated, **not** recommended]
+
+Instead of specifying strides by domain, one may choose to specify the strides
+for input and output data sequences. Let
+:math:`s^{\text{x}}_{j}, \ j \in \lbrace 0, 1, \ldots, d\rbrace` be the stride
+values for input (resp. output) data sequences if :math:`\text{x} = \text{i}`
+(resp. :math:`\text{x} = \text{o}`). Such
+:math:`s^{\text{x}}_0, s^{\text{x}}_1, \dots, s^{\text{x}}_d` values may be
+communicated as elements, in that order, of a :math:`(d+1)`-long
+``std::vector<std::int64_t>`` configuration value, set for the (deprecated)
+configuration parameter ``config_param::INPUT_STRIDES`` if
+:math:`\text{x} = \text{i}` (resp. ``config_param::OUTPUT_STRIDES`` if
+:math:`\text{x} = \text{o}`).
+
+The values of :math:`s^{\text{i}}_{j}` and :math:`s^{\text{o}}_{j}` are to be
+used and considered by oneMKL if and only if
+:math:`s^{\text{fwd}}_{j} = s^{\text{bwd}}_{j} = 0, \forall j \in \lbrace 0, 1, \ldots, d\rbrace`.
+In such a case, :ref:`descriptor<onemkl_dft_descriptor>` objects must consider
+the data layouts corresponding to the two compute directions separately. As
+detailed above, relevant data sequence entries are accessed as element of data
+containers (``sycl::buffer`` objects or device-accessible USM allocations)
+provided to the compute function, the base data type of which is (possibly
+implicitly re-interpreted) as documented in :ref:`this
+table<onemkl_dft_config_data_implicitly_assumed_elementary_data_type>`. If using
+input and output strides, for any :math:`m` and multi-index
+:math:`\left(k_1, k_2, \ldots, k_d\right)` within :ref:`valid
+range<onemkl_dft_elementary_range_of_indices>`, the index to be used when
+accessing a data sequence entry - or part thereof - in forward domain is
+
+.. math::
+    s^{\text{x}}_0 + k_1\ s^{\text{x}}_1 + k_2\ s^{\text{x}}_2 + \dots + k_d\ s^{\text{x}}_d + m\ l^{\text{fwd}}
+
+where :math:`\text{x} = \text{i}` (resp. :math:`\text{x} = \text{o}`) for
+forward (resp. backward) DFT(s). Similarly, the index to be used when accessing
+a data sequence entry - or part thereof - in backward domain is
+
+.. math::
+    s^{\text{x}}_0 + k_1\ s^{\text{x}}_1 + k_2\ s^{\text{x}}_2 + \dots + k_d\ s^{\text{x}}_d + m\ l^{\text{bwd}}
+
+where :math:`\text{x} = \text{o}` (resp. :math:`\text{x} = \text{i}`) for
+forward (resp. backward) DFT(s).
+
+As a consequence, configuring :ref:`descriptor<onemkl_dft_descriptor>` objects
+using these deprecated configuration parameters makes their configuration
+possibly direction-dependent (if different stride values are to be used in
+forward and backward domains). Since the intended compute direction is unknown
```

Comment:
```suggestion
using these deprecated configuration parameters makes their configuration
direction-dependent when different stride values are used in
forward and backward domains. Since the intended compute direction is unknown
```